tensorboard/webapp/core/effects/core_effects_test.ts

```diff
@@ -462,6 +462,20 @@ describe('core_effects', () => {
         polymerRunsFetchRequested(),
       ]);
 
+      // Ensure Alias Number changes invoke another request.
+      store.overrideSelector(getExperimentIdToExperimentAliasMap, {
+        eid1: {aliasText: 'alias 1', aliasNumber: 2},
+        eid2: {aliasText: 'alias 2.1', aliasNumber: 1},
+      });
+      store.refreshState();
+      tick(TEST_ONLY.ALIAS_CHANGE_RUNS_RELOAD_THROTTLE_IN_MS * 2);
+      expect(recordedActions).toEqual([
+        polymerRunsFetchRequested(),
+        polymerRunsFetchRequested(),
+        polymerRunsFetchRequested(),
+        polymerRunsFetchRequested(),
+      ]);
+
       discardPeriodicTasks();
```

Comment:
If it is not troublesome, would you mind factoring this test out into its own behavioral test spec? This spec was already scary long and it is even longer now.

Reply:
No problem. I decided to copy the logic instead of making a separate describe/before each. Seemed unnecessarily abstract for just 2 tests.